components/common/ConnectWallet/WalletAssetNfts.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <div v-if="nfts.length">
+    <hr class="my-4" />
+    <p class="has-text-grey is-size-7 mb-2">Recent NFTs</p>
+    <div class="nfts">
+      <a
+        v-for="nft in nfts.slice(0, 3)"
+        :key="nft.meta.id"
+        :href="`/${urlPrefix}/gallery/${nft.id}`">
+        <MediaItem
+          :src="sanitizeIpfsUrl(nft.meta.image)"
+          :mime-type="nft.type" />
+      </a>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { MediaItem } from '@kodadot1/brick'
+import { sanitizeIpfsUrl } from '@/utils/ipfs'
+import { getNftMetadata } from '@/composables/useNft'
+
+import type { NFTWithMetadata } from '@/composables/useNft'
+import { getMimeType } from '~~/utils/gallery/media'
+
+const { accountId } = useAuth()
+const { client, urlPrefix } = usePrefix()
+
+const queryPrefix = client.value === 'ksm' ? 'chain-ksm' : client.value
+const { data } = useGraphql({
+  queryPrefix,
+  queryName: 'nftListWithSearch',
+  variables: {
+    first: 10,
+    search: [
+      {
+        currentOwner_eq: accountId.value,
+      },
+    ],
+  },
+})
```

Comment:
composable?

looks pretty similar to ```useItemsGrid```

Reply:
yes similar, at first I wanna use `useItemsGrid` but that composable need some required params